feature/bgp/bgp.go

```diff
@@ -46,6 +46,12 @@ func (b *BGP) WithAS(as uint32) *BGP {
 	return b
 }
 
+// WithAfiSafiGlobal sets the AFI-SAFI type for BGP global.
+func (b *BGP) WithAfiSafiGlobal(name oc.E_BgpTypes_AFI_SAFI_TYPE) *BGP {
```

Comment:
WithAFISAFI() This would match the methods in Neighbor and PeerGroup.

Reply:
Changed it to WithAFISAFI() itself. Done!